pypeit/scripts/install_ql_calibs.py

```diff
@@ -5,8 +5,10 @@
 .. include:: ../include/links.rst
 """
 
+# TODO: Do people actually use this script?  The directory on the Google drive
```

Comment:
Maybe pose this question on the Slack. I have certainly never used it, and the QL stuff was changed recently, so maybe it's not needed?

Reply:
Added  #1786